src/robot/parsing/populators.py

```diff
@@ -160,10 +160,10 @@ def _directory_is_included(self, path, incl_suites):
         name = os.path.basename(os.path.normpath(path))
         return self._is_in_included_suites(name, incl_suites)
```

Comment:
Should't inclusion be checked against both the normal name and the long name also here?

Reply:
I didn't find a case that would benefit from this, so I don't think so.